README.md

```diff
@@ -55,6 +55,24 @@ pnpm exec biome --help
 
 To check types, you can run the `pnpm typecheck` command as other commands ignore types.
 
+## Tests
+
+The unit tests are written with [node:test](https://nodejs.org/api/test.html) and [node:assert](https://nodejs.org/api/assert.html).
+
+The unit tests can be run with the following command:
+
+```shell
+pnpm test
+```
+
+The end-to-end tests are written with [playwright](https://playwright.dev/).
+The tests are in `apps/class-solid/tests/*.spec.ts` and can be run with the following command:
+
+```shell
+cd ./apps/class-solid
+pnpm exec playwright test
```

Comment:
I had to `pnpm exec playwright install` first

Reply:
Did it tell you do that in a clear way? If not the docs should be improved.

Reply:
It was quite clear, yes, but I would still prefer to update the docs like so:

```suggestion
```shell
cd ./apps/class-solid
pnpm exec playwright install  # first time only
pnpm exec playwright test
```